### PR TITLE
Update limits

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review and merge.

### CODE REVIEW

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb [hunk 1]
```diff
@@ -65,14 +65,14 @@ def setup
...
-        :y_min => 34000, :y_max => 40000,
+        :y_min => 33000, :y_max => 41000,
         :historic => true
       },
       {
         :x => 'loadgiver',
         :y => 'feeder.throughput',
         :filter => {'loadgiver' => VESPA_JSON_FEEDER, 'size' => LARGE },
-        :y_min => 33000, :y_max => 38000,
+        :y_min => 39000, :y_max => 39000,
```
Comment (baldersheim) [inline tests/performance/programmatic_feed_client/programmatic_feed_client.rb:75]: min == max ?

**Review by baldersheim — DISMISSED** [review]

**Review by bjorncs — CHANGES_REQUESTED** [review]
See @baldersheim's comment

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb (hunk 1 above)
Comment (jonmv) [reply tests/performance/programmatic_feed_client/programmatic_feed_client.rb]: Yes, wee expect very consistent results for this ... "cough"

**Comment by jonmv** [comment]
PTAL.

**Review by bjorncs — APPROVED** [review]

**Pull request merged**